addons/web/static/src/core/dropdown/dropdown.js

```diff
@@ -286,11 +283,11 @@ export class Dropdown extends Component {
     }
 
     /**
-     * Opens the dropdown the mouse enters its toggler.
+     * Opens the dropdown when the mouse enters its toggler if the autoOpen prop is true.
      * NB: only if its siblings dropdown group is opened and if not a sub dropdown.
      */
     onTogglerMouseEnter() {
-        if (this.state.groupIsOpen && !this.state.open) {
+        if (this.props.autoOpen && this.state.groupIsOpen && !this.state.open) {
```

Comment:
How does it work with the following example ?
```xml
<Dropdown class="'A'" autoOpen="false"/>
<Dropdown class="'B'"/>
<Dropdown class="'C'"/>
```

With your diff, if you open A then hover on B or C they will open.

Maybe you should not do this and instead change the following line in `onDropdownStateChanged`
```diff
-this.state.groupIsOpen = args.newState.groupIsOpen;
+this.state.groupIsOpen = args.newState.groupIsOpen && this.props.autoOpen;
```

And also you should take into account `this.props.autoOpen` in `toggle()` and `open()`

With this suggestion the above scenario will act like this:
- if you open A, then B and C would not have their "groupOpen" to true,
- if you open B, then C would have its "groupOpen" set to true and A will not.

I think this is more consistent.

Reply:
noted and applied